x-pack/plugins/cases/public/components/add_comment/index.test.tsx

```diff
@@ -120,8 +119,8 @@ describe('AddComment ', () => {
       isLoading: true,
     }));
     const wrapper = mount(
-      <TestProviders>
-        <AddComment {...{ ...addCommentProps, userCanCrud: false }} />
+      <TestProviders permissions={{ all: false, read: false }}>
```

Comment:
Does the `read=false` affect the test in a way?

Reply:
No, `AddComment` is looking for ` {permissions.all && (`. I was just defining it to be complete.